x-pack/elastic-agent/pkg/agent/transpiler/steps_test.go

```diff
@@ -120,6 +125,8 @@ func TestExecFile_StdOut(t *testing.T) {
 }
 
 func TestExecFile_NoOutput(t *testing.T) {
+	t.Skip("skipping failing tests")
+
```

Comment:
@AndersonQ is this new? I haven't seen that in the original PR commits?

Reply:
that's a mistake, the're failing on master as well... let me fix it